modules/ROOT/pages/design-modify-raml-specs-conform.adoc

```diff
@@ -639,11 +667,33 @@ The parser would assume that `user` was the name of the example and that the obj
 
 ==== Including more than one named example in a single NamedExample fragment
 
+A NamedExample fragment can contain only one example. The following fragment would generate a warning:
+
 ----
+#%RAML 1.0 NamedExample
+Name1:
+   a: "b"
+   b: 1
 
+Name2:
+    a: "h"
+    b: 5
 ----
 
+To resolve the problem, you would need to split the fragment into two fragments:
```

Comment:
The user can split the fragment in two but remember that only one include is allowed from "examples" facet, so maybe it would be better to just delete the second example.